docs/templates/how-to-template.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Start with a gerund
```

Comment:
```suggestion
title: Start with a verb ending in -ing
```
I'd put that instead of gerund. Most non-english-majors or non-native speakers won't know what a gerund is.